doc/source/getting_started/about_odl.rst

```diff
@@ -158,29 +138,24 @@ right-hand side is 0.14, so we choose 0.1.
     >>> for i in range(niter):
     ...     residual = matrix_op(reco) - data
     ...     reco -= 0.1 * matrix_op.adjoint(residual)
-    ...     print(residual.norm())
-    1.41421356237
-    0.583095189485
-    0.240416305603
-    0.0991261822124
-    0.0408707719526
+    ...     print('{:.3}'.format(residual.norm()))
+    1.41
+    0.583
+    0.24
+    0.0991
+    0.0409
 
-If we now exchange ``matrix_op`` and ``data`` with a tomographic projector and line integral data,
-not a single line of code in the reconstruction method changes since the operator interface is
-exactly the same.
+If we now exchange ``matrix_op`` and ``data`` with a tomographic projector and line integral data, not a single line of code in the reconstruction method changes since the operator interface is exactly the same.
 
 
 Further features
 ================
 
 * A unified structure for representing tomographic acquisition geometries
-* Interfaces to fast external libraries, e.g. ASTRA for X-ray tomography, STIR for emission
-  tomography (preliminary), pyFFTW for fast Fourier transforms, ...
+* Interfaces to fast external libraries, e.g. ASTRA for X-ray tomography, STIR for emission tomography (preliminary), pyFFTW for fast Fourier transforms, ...
 * A growing number of "must-have" operators like `Gradient`, `FourierTransform`, `WaveletTransform`
-* Several solvers for variational inverse problems, ranging from simple gradient methods to
-  the Chambolle-Pock method (more to come...)
-* Standardized tests for the correctness of implementations of operators and spaces, e.g. does
-  the adjoint operator fulfill its defining relation?
+* Several solvers for variational inverse problems, ranging from simple gradient methods to the Chambolle-Pock method (more to come...)
```

Comment:
Chambolle-pock isnt our most complicated. I'd say something about "nonsmooth convex solvers such as (list a few)" also remove "more to come".